test_env/Dockerfile.client

```diff
@@ -1,13 +1,23 @@
 FROM ubuntu:22.04
 
-ENV RUST_LOG=info
+ARG LATENCY=0ms
+ARG RUST_LOG=info
 
-RUN apt update && apt install -y iputils-ping dnsutils
+ENV LATENCY=$LATENCY
+ENV RUST_LOG=$RUST_LOG
 
-COPY ./target/release /app
-CMD export SERVER_IP=$(dig +short relay_server -t a); echo $SERVER_IP; \
-    /app/examples/http_client \
-        --api-port 8081 \
-        --relay-addr tcp://$SERVER_IP:7464 \
-        --p2p-bind-addr tcp://0.0.0.0:9081
+RUN apt update \
+    && apt install -y iputils-ping dnsutils iproute2
 
+# Copy debug or release (prefered) build
+COPY ./target/debu[g]/example[s]/http_clien[t] /app/
+COPY ./target/releas[e]/example[s]/http_clien[t] /app/
```

Comment:
These `[x]` are a glob patterns to perform  `COPY` only if file exists.
Consecutive `test -f` checks if anything has been copied to image.

Reply:
I'm concerned about the debug/release irrelevance. Eventually it's not going to encourage using release for testing.

Reply:
It is only to not force me to do `release` build every time I need to test some changes locally.

Reply:
But if you have ever built a release version then this would not work - COPY would overwrite the newer debug version with older release.

Reply:
I am aware of it. I did it this way, because it was more convenient than passing extra parameters through docker-compose, scripts, tests, GH action files, and cmdline calls. If it will be a problem we can change it later.